Resources/Prototypes/Entities/Objects/Devices/encryption_keys.yml

```diff
@@ -0,0 +1,272 @@
+#EncryptionKeyComponent, EncryptionKeySystem
+- type: entity
+  abstract: true
+  parent: BaseItem
+  id: EncryptionKey
+  name: encryption key
+  description: A small cypher chip for headsets.
+  components:
+  - type: Item
+    sprite: Objects/Devices/encryption_keys.rsi
+  - type: EncryptionKey
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyCommon
+  name: passanger encryption key
+  description: An encryption key used by anyone.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyCargo
+  name: cargo encryption key
+  description: An encryption key used by supply employees.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Supply
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: cargo_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyQM
+  name: qm encryption key
+  description: An encryption key used by the boss of cargo.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Supply
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: qm_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyCentCom
+  name: central command encryption key
+  description: An encryption key used by captain's bosses.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - CentCom
+    - Engineering
+    - Medical
+    - Science
+    - Security
+    - Service
+    - Supply
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: bin_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyCommand
+  name: command encryption key
+  description: An encryption key used by crew's bosses.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Engineering
+    - Medical
+    - Science
+    - Security
+    - Service
+    - Supply
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: com_cypherkey
```

Comment:
Shouldn't this be EncryptionKeyCaptain? Command roles by themselves don't have access to these channels, only Captain.

Reply:
This channels was on HeadsetCommand, so yes, it should be Command, i don't know

Reply:
@Veritius this is the same key the HoP uses, so it should be EncryptionKeyCommand